js/src/personal/bryan/TimeDisplay.tsx

```diff
@@ -42,29 +52,33 @@ export function TimeDisplay() {
     }
     return String(hour)
   }, [date])
-  //rewrite away from function
   const Minutes = useMemo(() => {
     const minute = date.getMinutes()
     if (minute < 10) {
       return '0'.concat(String(minute))
     }
     return String(minute)
   }, [date])
-
   const AM = useMemo(() => {
     if (date.getHours() < 12) return 'AM'
     return 'PM'
   }, [date])
+  const Colon = useMemo(() => {
+    if (date.getSeconds() % 2 === 0) return ' '
```

Comment:
This is probably overkill for using a useMemo.

You could also just one line this by using a ternary.
`date.getSeconds() % 2 === 0 ?  '  ' :  ':'`